hr_employee_firstname/static/description/index.html

```diff
@@ -1,4 +1,3 @@
-<?xml version="1.0" encoding="utf-8"?>
```

Comment:
This file is autogenerated.

Could you check if that line is removed by running the README generator, please?

You can read instructions on how to run it [here](https://github.com/OCA/maintainer-tools/?tab=readme-ov-file#readme-generator).

Reply:
Thanks! Done. Could you check it, please?